do_mpc/model.py

```diff
@@ -798,6 +803,10 @@ def set_variable(self, var_type, var_name, shape=(1,1)):
         getattr(self, var_type)['var'].append(var)
         getattr(self, var_type)['name'].append(var_name)
 
+        # Fill integer list with input variables that are integer variables
```

Comment:
Better comment:

Update list of integer_inputs with the name of the current input.